corehq/apps/hqcase/api/core.py

```diff
@@ -14,6 +15,9 @@ def serialize_case(case):
         "date_opened": _isoformat(case.opened_on),
         "last_modified": _isoformat(case.modified_on),
         "server_last_modified": _isoformat(case.server_modified_on),
+        # This is used for cases that were just created, which haven't yet been indexed
+        # Providing this maintains compatibility with the es response
+        "indexed_on": _isoformat(datetime.utcnow()),
```

Comment:
I'm also tempted to include the field, but set to `null`, which seems more accurate - will take discussion to the [ticket](https://dimagi-dev.atlassian.net/browse/USH-2319)